src/cryptography/x509/__init__.py

```diff
@@ -51,6 +51,7 @@
 OID_FRESHEST_CRL = ExtensionOID.FRESHEST_CRL
 OID_INHIBIT_ANY_POLICY = ExtensionOID.INHIBIT_ANY_POLICY
 OID_ISSUER_ALTERNATIVE_NAME = ExtensionOID.ISSUER_ALTERNATIVE_NAME
+OID_ISSUING_DISTRIBUTION_POINT = ExtensionOID.ISSUING_DISTRIBUTION_POINT
```

Comment:
Are we exposing these? I thought this was just for backwards compat, and new ones should be on `ExtensionOID` only?